game/src/js/objects/characters/Character.js

```diff
@@ -50,6 +51,8 @@ export default class Character extends Phaser.Physics.Arcade.Sprite {
     this.setVelocity(0, 0);
     let projectile = new Projectile(this.scene, this.x, this.y, this.projectileType, targetX, targetY);
     this.scene.projectiles.add(projectile);
+    projectile.body.isCircle = true;
+    projectile.body.setCircle(40,-5,-5);
```

Comment:
I wonder how well this circle will work for other types of projectiles. Have you tested it with anything other than 'orb' by any chance?